styles.css

```diff
@@ -933,10 +949,10 @@ footer {
   }
 
   hr {
-    position: relative;
-    top: 0;
-    left: 0;
-    width: 100%;
+    position: relative !important;
+    width: 100% !important;
+    bottom: 0 !important;
+    left: 0 !important;
     border: 1px solid #212121;
   }
```

Comment:
- Hey, great job so far, however, there are some issues you need to work on in order to have your project approved. Please, kindly see these suggestions as an opportunity to improve your development skill. Kindly fix all the issues that would be pointed out below:

   - Please, kindly add some spaces to the h3 to be close to the Figma design as much as you can.
   - According to the Microverse list for [HTML & CSS requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), You have overused the `!important` rule in your CSS style definitions which is not allowed and not a good practice, except you are using some CSS framework to add your styles. As this appears to be in multiple lines in your CSS style definitions, please, kindly ***REMOVE ALL THE `!important` RULE FROM YOUR CODEBASE*** and use another approach to add the desired rule you want to meet this part of the CSS requirement. :pray: 
   - 


*Figma design*
![image](https://github.com/Xghayor/Portfolio/assets/59397159/247e9db2-0c28-415c-8ffd-a5fe2705d2fd)


*Your design*
![image](https://github.com/Xghayor/Portfolio/assets/59397159/e2206965-d9b7-4bb9-bcbd-b4fb01c4545e)